src/v0/util/index.js

```diff
@@ -1746,6 +1746,8 @@ const validatePhoneWithCountryCode = (phone) => {
   const regex = /^\+(?:[\d{] ?){6,14}\d$/;
   return !!regex.test(phone);
 };
+const isEmail = inputString => validateEmail(inputString)
```

Comment:
Let's change `validateEmail` itself 😄 . And work on the function renaming in some other PR

Reply:
Ok

Reply:
Done